DQM/BeamMonitor/plugins/OnlineBeamMonitor.cc

```diff
@@ -199,6 +213,7 @@ std::shared_ptr<onlinebeammonitor::NoCache> OnlineBeamMonitor::globalBeginLumino
   }
   if (auto bsTransientHandle = iSetup.getHandle(bsTransientToken_)) {
     auto const& spotDB = *bsTransientHandle;
+    std::cout <<" from the DB "<<spotDB<<std::endl;
```

Comment:
Can you remove the cout (or replace it with `edm::Log`)?

Reply:
i forgot to remove it, doing it now (it was just a test)